app/controllers/RegimeAPIController.scala

```diff
@@ -57,41 +59,39 @@ class RegimeAPIController @Inject()(auditService: AuditService,
     implicit request => {
       val agnosticEnrolmenKey = AgnosticEnrolmentKey(regime, idType, id)
-      // composeEnrolmentKey(regime, id).andThen { enrolmentKey =>
-        // import enrolmentKey._
         getPenaltyDetailsService.getDataFromPenaltyService(agnosticEnrolmenKey).flatMap {
           _.fold({
-            case PenaltyDetailsParser.GetPenaltyDetailsFailureResponse(status) if status == BAD_REQUEST => {
+            case PenaltyDetailsParser.PenaltyDetailsFailureResponse(status, _) if status == BAD_REQUEST => {
               //logger.info(s"[RegimeAPIController][getSummaryDataForVRN] - 1812 call (VATVC/BTA API) returned $status for VRN: $vrn")
               Future(NotFound(s"A downstream call returned 400 for ${agnosticEnrolmenKey.idType.value}: ${agnosticEnrolmenKey.id.value}"))
             }
-            case PenaltyDetailsParser.GetPenaltyDetailsFailureResponse(status) if status == NOT_FOUND => {
+            case PenaltyDetailsParser.PenaltyDetailsFailureResponse(status, _) if status == NOT_FOUND => {
               //logger.info(s"[RegimeAPIController][getSummaryDataForVRN] - 1812 call (VATVC/BTA API) returned $status for VRN: $vrn")
               Future(NotFound(s"A downstream call returned 404 for ${agnosticEnrolmenKey.idType.value}: ${agnosticEnrolmenKey.id.value}"))
             }
-            case PenaltyDetailsParser.GetPenaltyDetailsFailureResponse(status) if status == UNPROCESSABLE_ENTITY => {
+            case PenaltyDetailsParser.PenaltyDetailsFailureResponse(status, _) if status == UNPROCESSABLE_ENTITY => {
               //Temporary measure to avoid 422 causing issues
-              val responsePayload = GetPenaltyDetailsSuccessResponse(GetPenaltyDetails(totalisations = None, lateSubmissionPenalty = None, latePaymentPenalty = None, breathingSpace = None))
+              val responsePayload = PenaltyDetailsSuccessResponse(PenaltyDetails(Instant.now(), totalisations = None, lateSubmissionPenalty = None, latePaymentPenalty = None, breathingSpace = None))
               //logger.info(s"[RegimeAPIController][getSummaryDataForVRN] - 1812 call (VATVC/BTA API) returned $status for VRN: $vrn - Overriding response")
               Future(returnResponseForAPI(responsePayload.penaltyDetails, agnosticEnrolmenKey))
             }
-            case PenaltyDetailsParser.GetPenaltyDetailsFailureResponse(status) => {
+            case PenaltyDetailsParser.PenaltyDetailsFailureResponse(status, _) => {
               //logger.info(s"[RegimeAPIController][getSummaryDataForVRN] - 1812 call (VATVC/BTA API) returned an unexpected status: $status")
               Future(InternalServerError(s"A downstream call returned an unexpected status: $status for $agnosticEnrolmenKey"))
             }
-            case PenaltyDetailsParser.GetPenaltyDetailsMalformed => {
+            case PenaltyDetailsParser.PenaltyDetailsMalformed => {
               PagerDutyHelper.log("getSummaryDataForVRN", MALFORMED_RESPONSE_FROM_1812_API)
```

Comment:
Is there a reason each case has a commented logger? If we want to log at this level we could extract the log and response to be singular outside the match

Reply:
There's lot of other commented braces and things in this file, could generally be neatened up if these things aren't needed